### DISCUSSION TITLE
Changelog for 4.8.6

### DISCUSSION THREAD

**Discussion by timbru31** [telegraf/telegraf#1666] [Q&A] [answered]
Dependabot informed me that 4.8.6 is available, yet I see no CHANGELOG nor git tag.
Can you provide the missing information?

Thanks :)

**Comment by MKRhere** [comment, marked as answer]
There is no code change, only one line in the README. It was an emergency publish. The next release will be 4.9.0 due shortly later today.

**Reply by timbru31** [reply]
Thanks for clarification. 👍
